Scripts/ImportVirtualMachines/ImportVirtualMachines.ps1

```diff
@@ -0,0 +1,184 @@
+﻿param
+(
+    [Parameter(Mandatory=$true, HelpMessage="The Subscription Id that contains the source DevTest Lab for copying virtual machines")]
+    [string] $SourceSubscriptionId,
+
+    [Parameter(Mandatory=$true, HelpMessage="The name of the source DevTest Lab")]
+    [string] $SourceDevTestLabName,
+
+    [Parameter(HelpMessage="The name of the source Virtual Machine (located in the source DevTest Lab) to copy, if you omit this parameter all VMs will be copied")]
+    [string] $SourceVirtualMachineName,
+
+    [Parameter(HelpMessage="The name of the destination Virtual Machine in the case you want to use a different machine name when a SourceVirtualMachineName is specified")]
+    [string] $DestinationVirtualMachineName,
+
+    [Parameter(Mandatory=$true, HelpMessage="The Subscription Id that contains the destination DevTest Lab for copying virtual machines")]
+    [string] $DestinationSubscriptionId,
+
+    [Parameter(Mandatory=$true, HelpMessage="The name of the destination DevTest Lab")]
+    [string] $DestinationDevTestLabName
+)
+
+# -----------------------------------------------------------------
+# Function to select a subscription - we save a little time only switching if we're not already on that subscription
+# -----------------------------------------------------------------
+function SelectSubscription($subId){
+    # switch to another subscription assuming it's not the one we're already on
+    if((Get-AzureRmContext).Subscription.Id -ne $subId){
+        Write-Output "Switching to subscription $subId"
+        $result = Set-AzureRmContext -SubscriptionId $subId
+    }
+}
+
+# -----------------------------------------------------------------
+# Function to Save profile, need this for Powershell jobs to pass authorization information
+# -----------------------------------------------------------------
+function SaveProfile {
```

Comment:
Fixed - since the function turned into a one-liner, and is only used in one place, I just moved the one line to the call site and got rid of the function.